streaming/vision/convert/ade20k.py

```diff
@@ -153,14 +153,17 @@ def main(args: Namespace) -> None:
             raise ValueError(f'Number of samples in a dataset doesn\'t match. Expected ' +
                              f'{expected_num_samples}, but got {len(samples)}')
 
-        split_images_out_dir = os.path.join(args.out_root, split)
+        out_dir = os.path.join(args.out_root, split)
         hashes = get_list_arg(args.hashes)
 
         if args.progbar:
             samples = tqdm(samples, leave=args.leave)
 
-        with MDSWriter(split_images_out_dir, fields, args.compression, hashes,
-                       args.size_limit) as out:
+        with MDSWriter(local=out_dir,
```

Comment:
nit: `split_dir` in sync with others ?

Reply:
here we have `in_root`, `out_root`, `split`, so `split_dir` would be confusing